data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/JsonTypeDefinitionBeaconRestApi.java

```diff
@@ -13,15 +13,236 @@
+import tech.pegasys.teku.validator.api.NodeSyncingException;
+import tech.pegasys.teku.validator.coordinator.Eth1DataProvider;
+
 public class JsonTypeDefinitionBeaconRestApi implements BeaconRestApi {
+  private final RestApi restApi;
+
+  public JsonTypeDefinitionBeaconRestApi(
+      final BeaconRestApiConfig config,
+      final DataProvider dataProvider,
+      final Eth1DataProvider eth1DataProvider,
+      final Spec spec) {
+    restApi = create(config, dataProvider, eth1DataProvider, spec);
+  }
 
   @Override
   public void start() {
-    throw new UnsupportedOperationException("Unsupported start operation");
+    restApi.start().ifExceptionGetsHereRaiseABug();
   }
 
   @Override
   public void stop() {
-    throw new UnsupportedOperationException("Unsupported stop operation");
+    restApi.stop().ifExceptionGetsHereRaiseABug();
+  }
+
+  private static RestApi create(
+      final BeaconRestApiConfig config,
+      final DataProvider dataProvider,
+      final Eth1DataProvider eth1DataProvider,
+      final Spec spec) {
+    final SchemaDefinitionCache schemaCache = new SchemaDefinitionCache(spec);
+    return new RestApiBuilder()
+        .openApiInfo(
+            openApi ->
+                openApi
+                    .title(StringUtils.capitalize(VersionProvider.CLIENT_IDENTITY))
+                    .version(VersionProvider.IMPLEMENTATION_VERSION)
+                    .description(
+                        "A minimal API specification for the beacon node, which enables a validator "
+                            + "to connect and perform its obligations on the Ethereum beacon chain.")
+                    .license("Apache 2.0", "https://www.apache.org/licenses/LICENSE-2.0.html"))
+        .openApiDocsEnabled(config.isRestApiDocsEnabled())
+        .listenAddress(config.getRestApiInterface())
+        .port(config.getRestApiPort())
+        .maxUrlLength(config.getMaxUrlLength())
+        .corsAllowedOrigins(config.getRestApiCorsAllowedOrigins())
+        .hostAllowlist(config.getRestApiHostAllowlist())
+        .exceptionHandler(
+            ChainDataUnavailableException.class, (throwable) -> HttpErrorResponse.noContent())
+        .exceptionHandler(
+            NodeSyncingException.class, (throwable) -> HttpErrorResponse.serviceUnavailable())
+        .exceptionHandler(
+            ServiceUnavailableException.class,
+            (throwable) -> HttpErrorResponse.serviceUnavailable())
+        .exceptionHandler(
+            ContentTypeNotSupportedException.class,
+            (throwable) -> new HttpErrorResponse(SC_UNSUPPORTED_MEDIA_TYPE, throwable.getMessage()))
+        .exceptionHandler(
+            BadRequestException.class,
+            (throwable) -> HttpErrorResponse.badRequest(throwable.getMessage()))
+        .exceptionHandler(
+            JsonProcessingException.class,
+            (throwable) -> HttpErrorResponse.badRequest(throwable.getMessage()))
+        .exceptionHandler(
```

Comment:
We need to add a new handler for `MissingDepositsException` in here (it was only added to the old setup while you were working on this PR).